### PR TITLE
Send 403 forbidden when a user does not have permission to delete the…

### PR DESCRIPTION
**Pull request by shriramshankar**
… association

### CODE REVIEW

File: utils/jwtUtil.js [hunk 1]
```diff
@@ -12,21 +12,34 @@
...
+ * @param {Error} err - The error to handle
+ * @param {String} modelName - The DB model name, used to disambiguate field
+ *  names
+ */
+function handleError(next, err, modelName) {
+  err.resource = modelName;
+  next(err);
+}
+
+/**
```
Comment (shriramshankar) [inline utils/jwtUtil.js]: created another copy of handleError here to handle the circular reference issue.

**Comment by shriramshankar** [comment]
@annyhe @iamigo My other PR that depends on this is ready to be created. Can we merge this?

**Comment by iamigo** [comment]
waiting for @annyhe review

**Comment by iamigo** [comment]
(or if @annyhe is not available feel free to ask @pallavi2209)

**Comment by shriramshankar** [comment]
@pallavi2209 can you please help with this review if you are free?

**Comment by annyhe** [comment]
@shriramshankar Looks good.

**Pull request merged**